requirements/dev.txt

```diff
@@ -7,7 +7,7 @@ freezegun>=0.3.12,<0.4.0
 Jinja2>=2.8,<3
 mock>=2.0.0,<2.1.0
 moto>=1.3.8,<2.0.0
-networkx==2.1
+networkx>=2.5
```

Comment:
how about we scope to less than 3 as well?  `networkx>=2.5,<3`?

Reply:
good idea